app/src/androidTest/java/com/sdp/movemeet/view/navigation/security/ProfileActivitySecurityTest.java

```diff
@@ -20,12 +21,13 @@
 
     @Before
     public void checkUserNotLogged() {
+        ProfileActivity.enableNav = false;
         FirebaseAuth firebaseAuth = FirebaseAuth.getInstance();
         if (firebaseAuth.getCurrentUser() != null) firebaseAuth.signOut();
     }
 
-    @Rule
-    public ActivityScenarioRule<ProfileActivity> testRule = new ActivityScenarioRule<>(ProfileActivity.class);
```

Comment:
Why can we get rid of this ActivityScenarioRule here? Don't we have to specify that we have to test the ProfileActivity somewhere in this test?

Reply:
With the new Navigation, all navigation tests are being redone so everything has been commented here for now.